astropy/tests/runner.py

```diff
@@ -280,6 +280,46 @@ class TestRunner(TestRunnerBase):
     A test runner for astropy tests
     """
 
+    def packages_path(self, packages, base_path, error=None, warning=None):
+        """
+        Generates the path for multiple packages.
+
+        Parameters
+        ----------
+        packages : str
+            Comma separated string of packages.
+        base_path : str
+            Base path to the source code or documentation.
+        error : str
+            Error message to be raised as ``ValueError``. Individual package
+            name and path can be accessed by ``{0}`` or ``{1}``
```

Comment:
This is getting to ridiculous detail, but you could use the names and let them be accessed by `{name}` and `{path}`...